crates/sui-indexer/src/metrics.rs

```diff
@@ -16,7 +16,6 @@ const LATENCY_SEC_BUCKETS: &[f64] = &[
 
 #[derive(Clone)]
 pub struct IndexerCheckpointHandlerMetrics {
-    pub total_checkpoint_requested: IntCounter,
```

Comment:
with parallel downloading, we constantly over fetch, so this metric is no longer informative.